error: no matching function for call to ‘DBusInt::send(std::__cxx11::string)’

I have debian 10 with NVIDIA driver + cuda.

I run:
```
git clone https://github.com/ethereum-mining/ethminer && cd ethminer
git submodule update --init --recursive
mkdir build
cd build
cmake .. -DETHDBUS=ON -DBINKERN=ON -DAPICORE=ON -DETHASHCUDA=ON -DETHASHCL=OFF
cmake --build .
```

And got error:
**ethminer/ethminer/main.cpp:115:53: error: no matching function for call to ‘DBusInt::send(std::__cxx11::string)’
             dbusint.send(Farm::f().Telemetry().str());**

Full cmake --build . report here:
```
root@debian:/usr/local/src/ethminer/build# cmake --build .
Scanning dependencies of target ethminer-buildinfo-git
[  0%] Built target ethminer-buildinfo-git
[  3%] Updating ethminer-buildinfo:
       Project Version:  0.19.0-alpha.0 (prerelease)
       System Name:      linux
       System Processor: x86_64
       Compiler ID:      gnu
       Compiler Version: 8.3.0
       Build Type:       release
       Git Info:         0.19.0-alpha.0 0 b4919aeb6ddcf833c2906b3001898dd8d0358427
       Timestamp:        2019-08-27T14:48:27
Scanning dependencies of target ethminer-buildinfo
[  6%] Building C object CMakeFiles/ethminer-buildinfo.dir/ethminer/buildinfo.c.o
[ 10%] Linking C static library ethminer/libethminer-buildinfo.a
[ 10%] Built target ethminer-buildinfo
Scanning dependencies of target devcore
[ 13%] Building CXX object libdevcore/CMakeFiles/devcore.dir/CommonData.cpp.o
[ 16%] Building CXX object libdevcore/CMakeFiles/devcore.dir/FixedHash.cpp.o
[ 20%] Building CXX object libdevcore/CMakeFiles/devcore.dir/Log.cpp.o
[ 23%] Building CXX object libdevcore/CMakeFiles/devcore.dir/Worker.cpp.o
[ 26%] Linking CXX static library libdevcore.a
[ 26%] Built target devcore
Scanning dependencies of target hwmon
[ 30%] Building CXX object libhwmon/CMakeFiles/hwmon.dir/wraphelper.cpp.o
[ 33%] Building CXX object libhwmon/CMakeFiles/hwmon.dir/wrapnvml.cpp.o
[ 36%] Building CXX object libhwmon/CMakeFiles/hwmon.dir/wrapadl.cpp.o
[ 40%] Building CXX object libhwmon/CMakeFiles/hwmon.dir/wrapamdsysfs.cpp.o
[ 43%] Linking CXX static library libhwmon.a
[ 43%] Built target hwmon
[ 46%] Building NVCC (Device) object libethash-cuda/CMakeFiles/ethash-cuda.dir/ethash-cuda_generated_ethash_cuda_miner_kernel.cu.o
ptxas info    : 0 bytes gmem, 264 bytes cmem[3]
ptxas info    : Compiling entry function '_Z25ethash_calculate_dag_itemj' for 'sm_30'
ptxas info    : Function properties for _Z25ethash_calculate_dag_itemj
    480 bytes stack frame, 696 bytes spill stores, 460 bytes spill loads
ptxas info    : Used 63 registers, 324 bytes cmem[0], 36 bytes cmem[2]
ptxas info    : Compiling entry function '_Z13ethash_searchPV14Search_resultsm' for 'sm_30'
ptxas info    : Function properties for _Z13ethash_searchPV14Search_resultsm
    80 bytes stack frame, 196 bytes spill stores, 176 bytes spill loads
ptxas info    : Used 63 registers, 336 bytes cmem[0], 32 bytes cmem[2]
ptxas info    : 0 bytes gmem, 264 bytes cmem[3]
ptxas info    : Compiling entry function '_Z25ethash_calculate_dag_itemj' for 'sm_35'
ptxas info    : Function properties for _Z25ethash_calculate_dag_itemj
    272 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 88 registers, 324 bytes cmem[0]
ptxas info    : Compiling entry function '_Z13ethash_searchPV14Search_resultsm' for 'sm_35'
ptxas info    : Function properties for _Z13ethash_searchPV14Search_resultsm
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 84 registers, 336 bytes cmem[0]
ptxas info    : 0 bytes gmem, 264 bytes cmem[3]
ptxas info    : Compiling entry function '_Z25ethash_calculate_dag_itemj' for 'sm_50'
ptxas info    : Function properties for _Z25ethash_calculate_dag_itemj
    272 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 95 registers, 324 bytes cmem[0]
ptxas info    : Compiling entry function '_Z13ethash_searchPV14Search_resultsm' for 'sm_50'
ptxas info    : Function properties for _Z13ethash_searchPV14Search_resultsm
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 95 registers, 336 bytes cmem[0]
ptxas info    : 0 bytes gmem, 264 bytes cmem[3]
ptxas info    : Compiling entry function '_Z25ethash_calculate_dag_itemj' for 'sm_52'
ptxas info    : Function properties for _Z25ethash_calculate_dag_itemj
    272 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 95 registers, 324 bytes cmem[0]
ptxas info    : Compiling entry function '_Z13ethash_searchPV14Search_resultsm' for 'sm_52'
ptxas info    : Function properties for _Z13ethash_searchPV14Search_resultsm
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 95 registers, 336 bytes cmem[0]
ptxas info    : 0 bytes gmem, 264 bytes cmem[3]
ptxas info    : Compiling entry function '_Z25ethash_calculate_dag_itemj' for 'sm_53'
ptxas info    : Function properties for _Z25ethash_calculate_dag_itemj
    272 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 95 registers, 324 bytes cmem[0]
ptxas info    : Compiling entry function '_Z13ethash_searchPV14Search_resultsm' for 'sm_53'
ptxas info    : Function properties for _Z13ethash_searchPV14Search_resultsm
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 95 registers, 336 bytes cmem[0]
ptxas info    : 0 bytes gmem, 264 bytes cmem[3]
ptxas info    : Compiling entry function '_Z25ethash_calculate_dag_itemj' for 'sm_60'
ptxas info    : Function properties for _Z25ethash_calculate_dag_itemj
    272 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 88 registers, 324 bytes cmem[0]
ptxas info    : Compiling entry function '_Z13ethash_searchPV14Search_resultsm' for 'sm_60'
ptxas info    : Function properties for _Z13ethash_searchPV14Search_resultsm
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 87 registers, 336 bytes cmem[0]
ptxas info    : 0 bytes gmem, 264 bytes cmem[3]
ptxas info    : Compiling entry function '_Z25ethash_calculate_dag_itemj' for 'sm_61'
ptxas info    : Function properties for _Z25ethash_calculate_dag_itemj
    272 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 95 registers, 324 bytes cmem[0]
ptxas info    : Compiling entry function '_Z13ethash_searchPV14Search_resultsm' for 'sm_61'
ptxas info    : Function properties for _Z13ethash_searchPV14Search_resultsm
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 95 registers, 336 bytes cmem[0]
ptxas info    : 0 bytes gmem, 264 bytes cmem[3]
ptxas info    : Compiling entry function '_Z25ethash_calculate_dag_itemj' for 'sm_62'
ptxas info    : Function properties for _Z25ethash_calculate_dag_itemj
    272 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 95 registers, 324 bytes cmem[0]
ptxas info    : Compiling entry function '_Z13ethash_searchPV14Search_resultsm' for 'sm_62'
ptxas info    : Function properties for _Z13ethash_searchPV14Search_resultsm
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 95 registers, 336 bytes cmem[0]
ptxas info    : 0 bytes gmem, 264 bytes cmem[3]
ptxas info    : Compiling entry function '_Z25ethash_calculate_dag_itemj' for 'sm_70'
ptxas info    : Function properties for _Z25ethash_calculate_dag_itemj
    272 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 80 registers, 356 bytes cmem[0]
ptxas info    : Compiling entry function '_Z13ethash_searchPV14Search_resultsm' for 'sm_70'
ptxas info    : Function properties for _Z13ethash_searchPV14Search_resultsm
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 101 registers, 368 bytes cmem[0]
ptxas info    : 0 bytes gmem, 264 bytes cmem[3]
ptxas info    : Compiling entry function '_Z25ethash_calculate_dag_itemj' for 'sm_75'
ptxas info    : Function properties for _Z25ethash_calculate_dag_itemj
    272 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 80 registers, 356 bytes cmem[0]
ptxas info    : Compiling entry function '_Z13ethash_searchPV14Search_resultsm' for 'sm_75'
ptxas info    : Function properties for _Z13ethash_searchPV14Search_resultsm
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 100 registers, 368 bytes cmem[0]
Scanning dependencies of target ethash-cuda
[ 50%] Building CXX object libethash-cuda/CMakeFiles/ethash-cuda.dir/CUDAMiner.cpp.o
[ 53%] Linking CXX static library libethash-cuda.a
[ 53%] Built target ethash-cuda
Scanning dependencies of target ethcore
[ 56%] Building CXX object libethcore/CMakeFiles/ethcore.dir/EthashAux.cpp.o
[ 60%] Building CXX object libethcore/CMakeFiles/ethcore.dir/Farm.cpp.o
[ 63%] Building CXX object libethcore/CMakeFiles/ethcore.dir/Miner.cpp.o
[ 66%] Linking CXX static library libethcore.a
[ 66%] Built target ethcore
Scanning dependencies of target poolprotocols
[ 70%] Building CXX object libpoolprotocols/CMakeFiles/poolprotocols.dir/PoolURI.cpp.o
[ 73%] Building CXX object libpoolprotocols/CMakeFiles/poolprotocols.dir/PoolManager.cpp.o
[ 76%] Building CXX object libpoolprotocols/CMakeFiles/poolprotocols.dir/testing/SimulateClient.cpp.o
[ 80%] Building CXX object libpoolprotocols/CMakeFiles/poolprotocols.dir/stratum/EthStratumClient.cpp.o
[ 83%] Building CXX object libpoolprotocols/CMakeFiles/poolprotocols.dir/getwork/EthGetworkClient.cpp.o
[ 86%] Linking CXX static library libpoolprotocols.a
[ 86%] Built target poolprotocols
Scanning dependencies of target apicore
[ 90%] Building CXX object libapicore/CMakeFiles/apicore.dir/ApiServer.cpp.o
[ 93%] Linking CXX static library libapicore.a
[ 93%] Built target apicore
Scanning dependencies of target ethminer
[ 96%] Building CXX object ethminer/CMakeFiles/ethminer.dir/main.cpp.o
/usr/local/src/ethminer/ethminer/main.cpp: In member function ‘void MinerCLI::cliDisplayInterval_elapsed(const boost::system::error_code&)’:
/usr/local/src/ethminer/ethminer/main.cpp:115:53: error: no matching function for call to ‘DBusInt::send(std::__cxx11::string)’
             dbusint.send(Farm::f().Telemetry().str());
                                                     ^
In file included from /usr/local/src/ethminer/ethminer/main.cpp:71:
/usr/local/src/ethminer/ethminer/../ethminer/DBusInt.h:27:10: note: candidate: ‘void DBusInt::send(const char*)’
     void send(const char* hash)
          ^~~~
/usr/local/src/ethminer/ethminer/../ethminer/DBusInt.h:27:10: note:   no known conversion for argument 1 from ‘std::__cxx11::string’ {aka ‘std::__cxx11::basic_string<char>’} to ‘const char*’
make[2]: *** [ethminer/CMakeFiles/ethminer.dir/build.make:63: ethminer/CMakeFiles/ethminer.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:500: ethminer/CMakeFiles/ethminer.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
root@debian:/usr/local/src/ethminer/build#
```

--- a/ethminer/main.cpp
+++ b/ethminer/main.cpp
@@ -112,7 +112,7 @@ public:
             minelog << logLine;
 
 #if ETH_DBUS
-            dbusint.send(Farm::f().Telemetry().str());
+            dbusint.send(Farm::f().Telemetry().str().c_str());
 #endif
             // Resubmit timer
             m_cliDisplayTimer.expires_from_now(boost::posix_time::seconds(m_cliDisplayInterval));

